### DISCUSSION TITLE
How to add my own Security Code Scanning tool widget/extension?

### DISCUSSION THREAD

**Discussion by elangosenthilnathan** [community/community#86082] [New to GitHub]
When I navigate to https://github.com/orgID/repoID/actions/new?category=security I could see a bunch of security code scanning tools listed( for ex Fortify). How do I add my own security code scanning tool as an extension out there? Please guide me on building an extension and submitting it with GitHub.

**Comment by queenofcorgis** [comment]
@elangosenthilnathan hi there I'm getting a 404 when I try to access that page - do you have an updated link?
